.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+    "tabWidth": 4,
```

Comment:
This is cool!  I heard of this project but never had a chance to use it.

Does it actually re-format all the source files based on the style specified here?

Reply:
Yes, if you run it at the command line on the project. There's also a great Visual Studio Code extension for it at https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode. It's great for making sure all of the files are formatted consistently.